Robot Code/src/org/usfirst/frc/team3316/robot/subsystems/Hood.java

```diff
@@ -0,0 +1,49 @@
+package org.usfirst.frc.team3316.robot.subsystems;
+
+import org.usfirst.frc.team3316.robot.Robot;
+import org.usfirst.frc.team3316.robot.robotIO.DBugSpeedController;
+
+import edu.wpi.first.wpilibj.AnalogPotentiometer;
+
+public class Hood extends DBugSubsystemCC
+{
+	private DBugSpeedController hoodMotor;
+	private AnalogPotentiometer hoodPot;
+
+	public Hood()
+	{
+		hoodMotor = Robot.actuators.hoodMotor;
+		addSpeedController(hoodMotor);
+
+		hoodPot = Robot.sensors.hoodPot;
+	}
+
+	public void initDefaultCommand()
+	{
+	}
+
+	/**
+	 * Set the voltage for the motors of this subsystem.
+	 * @par v - The voltage for the motors (between 1.0 to -1.0).
+	 */
+	public boolean setMotors(double v)
+	{
+		// If the hood is able to move.
+		if (getAngle() > (double) config.get("hood_Pot_LowThresh")
```

Comment:
If by some chance we run out of the working range this doesn't allow us to fix this. The check should consider the direction we're trying to move the motor. ie:
If we're past top limit, moving down should be legal and vice versa.

perhaps we want to add an additional range in which you don't want to move at all because the mechanism might already be broken and you'll only make matters worse.